### ISSUE TITLE
make it easier to skip 10 seconds before or 10 seconds after

### ISSUE THREAD

**Issue by ldang** [socallinuxexpo/scale-av-cutter#41]
As a reviewer honing in on the right moment to nail down the start and end time, it would be nice to jump forward or backwards 10 seconds.   

Actual fine-tuning is done by just editing the timestamp directly.

This goes for capturing the thumbnail as well.

**Comment by destevae** [comment]
Do you mean jumping forwards and backwards on the Youtube video or the start/end/thumbnail times? Right now you can jump forward and backward 5 seconds on the Youtube video by pressing the left and right arrow keys.

**Comment by ldang** [comment]
@destevae: I just learned about the left and right arrow keys, so that would be nice to mention as a tool tip or help. 

But I will also note that it only works if I've interacted with the YouTube video.  If I've selected some of the radio buttons or done something else, the left/right arrow keys don't work. 

Maybe this issue could be handled with the following:
* Provide documentation about any keyboard shortcuts YouTube allows that makes it easy to skip forward and backward in the video.
* Provide some way where a keyboard shortcut would allow me to focus on the YouTube video.  Then I can just use whatever keyboard shortcuts YouTube allows.   Maybe that would involve a Javascript event listener or something.
